packages/perseus-core/src/data-schema.ts

```diff
@@ -642,7 +642,7 @@ export type GrapherAnswerTypes =
       };
 
 export type PerseusGrapherWidgetOptions = {
-    availableTypes: ReadonlyArray<
+    availableTypes: Array<
```

Comment:
For the sake of consistency, should we write these as:

```typescript 
    availableTypes: (
        | "absolute_value"
        | "exponential"
        | "linear"
        | "logarithm"
        | "quadratic"
        | "sinusoid"
        | "tangent"
     )[]
```

Reply:
I used `Array` here because I don't like holding readers in suspense unnecessarily. The problem with `(` is that when you see it, you don't know what's on the other end, which requires you to hold more in memory (and identify the matching `)`) as you're mentally parsing the code. By contrast, when you see `Array<` you know to expect exactly one type argument, and you know what that argument means.

Possibly relevant:
- https://github.com/benchristel/benchristel.github.io/wiki/FunctionalPipeline
- https://github.com/benchristel/benchristel.github.io/wiki/LongBranchLast